File: android/app/src/main/java/com/ex3/androidchat/models/contacts/ContactModel.java
package com.ex3.androidchat.models.contacts;

import java.time.LocalDateTime;

public class ContactModel {
    String id, name, server, last, profileImage;
    LocalDateTime lastdate;

    public void setId(String id) {
        this.id = id;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setServer(String server) {
        this.server = server;
    }

    public void setLast(String last) {
        this.last = last;
    }

    public void setProfileImage(String profileImage) {
        this.profileImage = profileImage;
    }

    public void setLastdate(LocalDateTime lastdate) {
        this.lastdate = lastdate;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getServer() {
        return server;
    }

    public String getLast() {
        return last;
    }

    public String getProfileImage() {
        return profileImage;
    }

    public LocalDateTime getLastdate() {
        return lastdate;
    }
}

File: Main/Models/Contacts/ContactModel.cs
namespace AspWebApi.Models.Contacts {
    public class ContactModel {
        public string Id { get; set; }
        public string ContactId { get; set; }

        public string Name { get; set; }

        public string Server { get; set; }

        public string Last { get; set; }

        public string Lastdate { get; set; }
        public string OfUser { get; set; }
        public string LastdateStr { get; set; }

        public string ProfileImage { get; set; }

        public ContactModel()
        {
        }

        public ContactModel(string id, string contactId, string name, string server, string last, string lastdate, string profileImage, string ofUser)
        {
            Id = id;
            ContactId = contactId;
            Name = name;
            Server = server;
            Last = last;
            Lastdate = lastdate;
            ProfileImage = profileImage;
            OfUser = ofUser;
            DateTime? dateTime;
            if (lastdate == null || lastdate == "")
            {
                dateTime = null;
                LastdateStr = null;
                return;
            }
            else
            {
                dateTime = Convert.ToDateTime(lastdate);
                LastdateStr = dateTime.Value.ToString("dd/MM/yyyy HH:mm");
            }
        }
    }
}
